src/components/KCheckbox/KCheckbox.vue

```diff
@@ -121,45 +143,139 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+/* Component mixins */
+
+@mixin kCheckboxIcon {
+  inset: 0;
+  left: calc(50% - 2.4px); // 2px is not enough, 3px is too much...
+  pointer-events: none;
+  position: absolute;
+  top: calc(50% + 1.75px); // 1px is not enough, 2px is too much...
+  transform: translate(-50%, -50%);
+  z-index: 1;
+}
```

Comment:
suggestion: can this go in the mixins file?

Reply:
I don't think this mixin will be reused - it's pretty specific to this component so I would leave it here.